rust/signed_doc/src/tests_utils/brand_parameters.rs

```diff
@@ -1,14 +1,17 @@
-use catalyst_signed_doc::providers::tests::TestCatalystProvider;
 use ed25519_dalek::ed25519::signature::Signer;
 
-use super::*;
+use crate::{
+    Builder, CatalystSignedDocument, ContentEncoding, ContentType, doc_types,
+    providers::tests::TestCatalystProvider, tests_utils::create_dummy_admin_key_pair, uuid::UuidV7,
+};
 
+/// # Errors
```

Comment:
This comment (and same comments everywhere) should either be removed or updated.